### PR TITLE
fix(be): add cursor and count scan params

### PR DESCRIPTION
**Pull request by noamilshtein**

### CODE REVIEW

File: src/types/RedisStore.interface.ts [hunk 1]
```diff
@@ -7,5 +7,5 @@ export interface RedisStore extends Store {
 
   getClient(): RedisClientType<RedisDefaultModules & RedisModules, RedisFunctions, RedisScripts>;
 
-  scan(pattern: string): Promise<ScanReply>;
+  scan(pattern: string, coursor? :number, count?: number): Promise<ScanReply>;
```
Comment (dmytro-baryshev) [inline src/types/RedisStore.interface.ts:10]:
```suggestion
  scan(pattern: string, cursor?:number, count?: number): Promise<ScanReply>;
```

**Review by Drumz0xF — DISMISSED** [review]

File: src/redis.ts [hunk 2]
```diff
@@ -106,9 +106,8 @@ class buildRedisStoreWithConfig implements RedisStore {
     return await this.redisCache.ttl(key);
   };
 
-  public async scan(pattern: string): Promise<ScanReply> {
-    let cursor = 0;
-    return await this.redisCache.scan(cursor, {MATCH: pattern});
+  public async scan(pattern: string, cursor: number = 0, count?: number): Promise<ScanReply> {
```
Comment (IAleks94) [inline src/redis.ts:109]:
```suggestion
  public async scan(pattern: string, cursor: number = 0, count = 10): Promise<ScanReply> {
```
let's use 10 as the default 
image

**Review by IAleks94 — COMMENTED** [review]
looks good

File: src/redis.ts (hunk 2 above)
Comment (noamilshtein) [reply src/redis.ts]: https://github.com/blindspot-security/node-cache-manager-redis-store/pull/11/commits/b8a12905ae5ee790cadc902d82b9213a6701a06e

**Review by ronbs — APPROVED** [review]

File: src/redis.ts (hunk 2 above)
Comment (IAleks94) [reply src/redis.ts]: thanks

**Pull request closed without merging**